src/app/containers/EpisodeList/RecentAudioEpisodes/index.jsx

```diff
@@ -124,7 +120,7 @@ const RecentAudioEpisodes = ({ masterBrand, episodes, brandId, pageType }) => {
         liProps={liProps}
       >
         {episodes.map(episode => (
-          <EpisodeList.Episode key={episode.id}>
+          <EpisodeList.Episode key={episode.id} viewTrackerRef={viewTrackerRef}>
```

Comment:
If you are to forward the ref I would just use the ref forwarding technique instead of creating a new viewTrackerRef prop.

Alternatively you could wrap EpisodeList.Episode in a div and put the ref on that

Reply:
👍 switched to `forwardRef` - there's a context provider wrapping `EpisodeList.Episode` so the ref needs some drilling

Reply:
ah ok that's unfortunate